predict.py

```diff
@@ -19,9 +20,13 @@
     StreamingTokenStopSequenceHandler,
 )
 
+from transformers import AutoTokenizer
```

Comment:
we should probably make sure we're using the rust tokenizer, which may require a tweak to the nix stuff. @nateraw what do we to do to get the fast tokenizer?

Reply:
I think it should work if you have `tokenizers` installed, which is registered as a `transformers` dep. Would be worthwhile to investigate.